docs/api/parquet.md

```diff
@@ -0,0 +1,30 @@
+# Parquet Format
```

Comment:
Let's also add a warning that deletes are currently dropped in the output stream.